composer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "buildempire/apartment",
+  "description": "Multi-tenancy for Lumen using PostgreSQL with Eloquent",
+  "keywords" : ["lumen", "postgres", "multitenancy"],
+  "type": "library",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Cameron Smith",
+      "email": "contact@buildempire.co.uk"
+    }
+  ],
+  "minimum-stability": "dev",
+  "require": {
+    "php": ">=5.6.4",
+    "laravel/lumen-framework": "5.4.*"
+  },
+  "version": "1.0.0"
```

Comment:
The key argument I made was that having the composer file use the "current" version (which at the moment is < 1.0.0 since unpublished) and adjusting the version just prior to publishing is the most robust solution (as opposed to the alternative of bumping the version to next immediately after publishing).

The reasoning here being that you only *really* know what kind of version bump you need (major/minor/patch etc.) immediately prior to a publish, so this is the point at which it makes most sense to alter the version number.  As such, this should be less that 1.0.0, so that it can be moved to 1.0.0 at the point of publishing - which would keep it consistent with later publishes.

I don't believe composer has an equivalent to `npm version`, but if it does, then having a version < 1.0.0 would be necessary in order to bump to 1.0.0 using such a command (`npm version` bawks in this scenario claiming you can't bump to the version it's already at).